src/layouts/ComponentLayout.astro

```diff
@@ -18,6 +21,7 @@ const { Content } = await render()
     <Prose>
       <Content />
     </Prose>
+    <NavigationButtons currentSlug={currentSlug} />
```

Comment:
what if there is a normal post where we do not want the naviagationbuttons, are they still visible or not?

Reply:
In the postlayout? Depending on if its defined in the sidebar but for example on a blog-like page not defined in the sidebar settings, no.